app/views/posts/index.html.erb

```diff
@@ -1 +1,31 @@
-<h1>Here your posts list with user_id </h1>
\ No newline at end of file
+<head>
+  <link rel="stylesheet" href="<%= asset_path('application.css') %>">
+</head>
+
+<div class= "photo-user">
+    <div class='user_img'>
+      <img src= <%= @user.photo %>  />
+    </div>
+<div class="user-card">
+<h1><%= @user.name  %></h1>
+<p>Number of posts:  <%= @user.posts_counter %> </p> 
+</div>
+</div>
+<% @user.posts.each_with_index do |post, index| %>
+<div class="post">
+<h2>Post- <%= link_to post.title, user_post_path(@user, post) %><br></h2>
+ <p> <%= post.text %>
+ </p>
+ <p>
+  Comments:  <%= post.comments_counter %> <br> Likes:  <%= post.likes_counter %>
+   </p>
```

Comment:
| Your Post Index | Wireframe Post Index |
| --- | --- |
| ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/6195bf1e-24e6-42ef-8a0d-ed9dee3a11eb) | ![image](https://github.com/otmanTR/BlogAppRoR/assets/93039359/72ffb7c3-09b8-4666-966e-f3a54e715820) |

- [ ] It would be nice if you make the image size and the content size similar as you can see in the wireframe to make your project follow the requirements :+1: but now your image size is a lot bigger than the expected size :fearful:
- [ ] It would be nice if you make the comments and likes counter at the right of the posts to match the wireframe :+1:
- [ ] It would be nice if you make show the comments instead of Comment 1 and Comment 2 because these are just placeholders in the wireframe to tell you that here you need to show the comments :smile:
- [ ] It would be nice if you make a Pagination button as you can see in the wireframe :+1:

Reply:
Thank you.